pom.xml

```diff
@@ -109,6 +111,52 @@
                     <scmCommentPrefix xml:space="preserve">MOJITO RELEASE: </scmCommentPrefix>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>2.24.1</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>apply</goal>
+                        </goals>
+                        <phase>compile</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <pom>
+                        <sortPom>
```

Comment:
I don't know about sortPom plugin, I thought it was nice but at the same time there were some things i wasn't a big fan of. For example it doesn't respect max line length in the prolog which a bit weird

Reply:
I can remove it from this PR - and we can re-consider adding/evaluating it as a separate change later on - there's plenty of changes without it already.